src/utils/serve-functions.js

```diff
@@ -154,14 +156,18 @@ function createHandler(dir) {
 
     const callback = createCallback(response)
     // we already checked that it exports a function named handler above
+    let envConfig = {}
+    if (fs.existsSync(path.resolve(path.dirname(functionPath)))) {
+      envConfig = dotenv.parse(fs.readFileSync(path.resolve(path.dirname(functionPath), '.env')))
```

Comment:
If the `.env` file does not exist, `fs.readFileSync()` will throw. Is this ok?

Reply:
We're checking for `fs.fs.existsSync` first and then calling `fs.readFileSync`

Reply:
We are checking if `{functionPath}` exists but not if `{functionPath}/.env` does.

Reply:
Oh good catch! Thanks